autometacal/python/moments.py

```diff
@@ -0,0 +1,68 @@
+from .tf_ngmix.moments import get_moments
+from .tf_ngmix.gmix import create_gmix,  fwhm_to_T
+from .tf_ngmix.pixels import make_pixels
+import tensorflow as tf
+
+def get_moment_ellipticities(images, scale, fwhm, **kwargs):
+  """
+  Gets ellipticity estimates from gaussian moments of stamps.
+  
+  Args:
+    images: A bach of images Tensor
+    scale: pixel scale
+    fwhm: full width at half maximum of the gaussian filter
+    centre_x, centre_y: centre of the image, if ommited, the centre pixel of the stamp is used.
+    weights: an image containing the weights of the
+    
+  Returns:
+    Gaussian-weighted moments: e1, e2 for the batch of images. according to the a+b/()
```

Comment:
I think this line is not complete? It's important that the documentation of this function tells us which ellipxlticity convention we are using

Reply:
@EiffL It is done, I think. I'll look into automated tests (Issue #8 ) now.